vault-plugins/flant_iam/model/roleresolver.go

```diff
@@ -31,14 +31,16 @@ type RoleInformer interface {
 type GroupInformer interface {
 	FindAllParentGroupsForUserUUID(TenantUUID, UserUUID) (map[GroupUUID]struct{}, error)
 	FindAllParentGroupsForServiceAccountUUID(TenantUUID, ServiceAccountUUID) (map[GroupUUID]struct{}, error)
+	FindAllSubjectsFor(TenantUUID, []UserUUID, []ServiceAccountUUID, []GroupUUID) (
+		map[UserUUID]struct{}, map[ServiceAccountUUID]struct{}, error)
 }
 
 type RoleBindingsInformer interface {
 	FindDirectRoleBindingsForTenantUser(TenantUUID, UserUUID) (map[RoleBindingUUID]*RoleBinding, error)
 	FindDirectRoleBindingsForTenantServiceAccount(TenantUUID, ServiceAccountUUID) (map[RoleBindingUUID]*RoleBinding, error)
 	FindDirectRoleBindingsForTenantGroups(TenantUUID, ...GroupUUID) (map[RoleBindingUUID]*RoleBinding, error)
 	FindDirectRoleBindingsForTenantProject(TenantUUID, ProjectUUID) (map[RoleBindingUUID]struct{}, error)
```

Comment:
Maybe this method should also return `map[RoleBindingUUID]*RoleBinding` as others?

Reply:
done